src/web/Content/client/dumpling.py

```diff
@@ -73,7 +73,7 @@ def Print(output):
             if(Output.s_logPath is not None and os.path.isfile(Output.s_logPath)):
                 # Note: The file must exist.
                 with open(Output.s_logPath, 'a') as log_file:
-                    log_file.write(output)
+                    log_file.write(output + '\n')
```

Comment:
You need to do work here to make sure that we are using the right line endings on Windows vs Linux. Probably want to use sys.platform and then use that to use the right line endings.

Reply:
```
os.linesep
The string used to separate (or, rather, terminate) lines on the current platform. This may be a single character, such as '\n' for POSIX, or multiple characters, for example, '\r\n' for Windows. Do not use os.linesep as a line terminator when writing files opened in text mode (the default); use a single '\n' instead, on all platforms.
```
As we are opening the file in the text mode I think this would be fine

Reply:
OK